DEPRECATED.md

```diff
@@ -0,0 +1,18 @@
+# Deprecated Features
+
+The following functionality of Firecracker is deprecated, and will be removed in
+a future major Firecracker release, in accordance with our
+[release policy](docs/RELEASE_POLICY.md).
+
+- \[[#2763](https://github.com/firecracker-microvm/firecracker/pull/2763)\] The
+  `vsock_id` body field in `PUT` requests on `/vsock`
+- \[[#2980](https://github.com/firecracker-microvm/firecracker/pull/2980)\] The
+  `mem_file_path` body field in `PUT` requests on `/snapshot/load`
+- \[[#2973](https://github.com/firecracker-microvm/firecracker/pull/2973)\]
+  MicroVM Metadata Service v1 (MMDSv1)
+- \[[#4126](https://github.com/firecracker-microvm/firecracker/pull/4126)\] The
```

Comment:
And static CPU templates in general also no? Those could be used if starting from a JSON file.

Reply:
Mh, no sure. @zulinx86 ?

Reply:
Yes, we'll drop the support of static CPU templates in general (not limited to API but also FC JSON config).

Reply:
Updated, thanks!